src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+import { PersonInfo } from '../components/PersonInfo';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [peopleLoading, setPeopleLoading] = useState(false);
+  const [peopleLoadingError, setPeopleLoadingError] = useState(false);
+
+  const { personId } = useParams();
+  const renderTable = !peopleLoading && people && people.length > 0;
+  const renderNoPeopleError = !peopleLoading && people && people.length === 0;
```

Comment:
Same here
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig